monitoring/microprofile-opentracing/src/test/java/io/quarkus/ts/microprofile/opentracing/MicroProfileIT.java

```diff
@@ -135,6 +135,6 @@ public void fallbackTest() {
     }
 
     protected int getAppPort() {
-        return app.getPort();
+        return app.getURI().getPort();
```

Comment:
I bet this won't do as there will be incorrect protocol. Have you run this test locally, please?

Reply:
Hmm, but we are only interested in port. I take it back, sorry.